packages/uilib/src/lib/plugins/communication-explorer/sidebar/sidebar.svelte

```diff
@@ -77,18 +95,31 @@
                     I don't think it adds much user value
```

Comment:
Is this comment still relevant regarding the fact that we are enhancing the select ?

Reply:
💯% correct and deleted.